network/benchmarks/netperf/launch.go

```diff
@@ -57,6 +57,7 @@ var (
 	tag            string
 	kubeConfig     string
 	netperfImage   string
+	cleanup_only   bool
```

Comment:
You should rename `cleanup_only` to `cleanupOnly` in order to make the linter happy.